lib/elixir/lib/file.ex

```diff
@@ -166,6 +166,11 @@ defmodule File do
   It can be regular file, directory, socket, symbolic link, named pipe or device file.
   Returns `false` for symbolic links pointing to non-existing targets.
 
+  ## Options
+
+  The list of options currently support a single option, the atom `:raw`,
```

Comment:
I think it should be: "The list of options currently support**s**"

Reply:
Thanks @fertapric and sorry for the type @mbuhot! I have pushed a fix!